### PR TITLE
Add support of docs

### PR DESCRIPTION
**Pull request by Sayam753**
Partially solves #4 
Just need to set a webhook to readthedocs.
How about adding a ANAME or AAAA record like docs.wildfirepy.org?

### CODE REVIEW

**Comment by Sayam753** [comment]
Can someone guide me how to pass tests?

**Comment by Raahul-Singh** [comment]
@masterashu Could you see why the welcome action isn't working?

**Comment by Raahul-Singh** [comment]
@Sayam753 We require a changelog with every PR we make.
go to `wildfirepy/changelog/` and make a file `47.feature.rst`
47 is the number of this PR.
What this PR is doing is adding a feature, the docs, in our case.
In that file, describe briefly what this PR is doing.
This helps us keep a track of stuff we are doing here.

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import os
+import sys
+sys.path.insert(0, os.path.abspath('..'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'WildfirePy'
+copyright = '2020, TBD'
+author = 'TBD'
```
Comment (Raahul-Singh) [inline docs/conf.py:22]:
```suggestion
copyright = '2020, The WildfirePy Project'
author = 'The WildfirePy Community'
```

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+# The full version, including alpha/beta/rc tags
+release = '0.x'
```
Comment (Raahul-Singh) [inline docs/conf.py:25]:
```suggestion
release = 'pre-alpha'
```

File: changelog/47.feature.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Added the support of docs to automate documentation builds, for every commit to master.
+
+Just need to set a webhook to readthedocs.
```
Comment (Raahul-Singh) [inline changelog/47.feature.rst:3]:
```suggestion

```
We only need the change info.
Adda an issue for the webhook.

Comment (Sayam753) [reply changelog/47.feature.rst:3]: Thanks for pointing out

**Comment by masterashu** [comment]
@Raahul-Singh Looks like the action is not having enough permissions required to comment on the repository hence it is failing. Same reason the missing changelog message was not commented.

**Comment by Raahul-Singh** [comment]
> @Raahul-Singh Looks like the action is not having enough permissions required to comment on the repository hence it is failing. Same reason the missing changelog message was not commented.

@ram-nad , @masterashu could you please figure this out?

**Comment by ram-nad** [comment]
@Raahul-Singh @masterashu It is by design made so. secrets are not available to PRs from the fork (due to security issues if the workflow file itself is modified by the PR). I think we need to make a bot for our organization to carry out these tasks.

**Comment by Raahul-Singh** [comment]
> @Raahul-Singh @masterashu It is by design made so. secrets are not available to PRs from the fork (due to security issues if the workflow file itself is modified by the PR). I think we need to make a bot for our organization to carry out these tasks.

I had this in mind. You wanna pick this up? I may find time next week to do this.

**Comment by ram-nad** [comment]
@Raahul-Singh sure. I will try but as I said before, setting up a bot is quite difficult, we need to manage server space, permissions, OAuth and a lot of other things. But it is worth a try. Can you open a new issue and assign it to me?

**Comment by Raahul-Singh** [comment]
> @Raahul-Singh sure. I will try but as I said before, setting up a bot is quite difficult, we need to manage server space, permissions, OAuth and a lot of other things. But it is worth a try. Can you open a new issue and assign it to me?

Cool. Look up baldrick from open astronomy

**Comment by ram-nad** [comment]
I already did. Quite different from what we will want. Moreover, it is in Python, making Github Apps using TypeScript is a lot easier.

**Comment by Raahul-Singh** [comment]
> I already did. Quite different from what we will want. Moreover, it is in Python, making Github Apps using TypeScript is a lot easier.

One less thing to fork then. :)

**Comment by Raahul-Singh** [comment]
We've updated the tests. MInd rebasing this?

**Comment by Sayam753** [comment]
Sure @Raahul-Singh

**Pull request merged**